tests/tracer/test_span.py

```diff
@@ -678,7 +678,3 @@ def get_exception_span(exception):
 
     exception_span = get_exception_span(Exception("DataDog/水"))
     assert "DataDog/水" == exception_span.get_tag(ERROR_MSG)
-
-    if six.PY3:
-        exception_span = get_exception_span(Exception("DataDog/水"))
-        assert "DataDog/水" == exception_span.get_tag(ERROR_MSG)
```

Comment:
Not sure of the context, but this block was PY3, not Py2, so it shouldn't be removed right?

Reply:
This logic appears to directly duplicate the two lines above it, so I can see how removing it makes sense.

Reply:
yeah, not sure why the assertions were run twice for Python 3.